types/store/draft.ts

```diff
@@ -22,10 +24,10 @@ export type PostDraft = {
     show?: boolean;
     remote?: boolean;
     metadata?: {
-        priority?: {
-            priority: PostPriority|'';
-            requested_ack?: boolean;
-            persistent_notifications?: boolean;
-        };
+        priority?: PostPriorityMetadata;
     };
 };
+
+export type NewPostDraft = PostDraft & {
+    content?: JSONContent;
```

Comment:
@michelengelen why do we need to introduce a new type if you add an optional key? I am not sure we are doing version bump for types, but we should update the existing `PostDraft.`
but would leave that decision with web platform cc// @hmhealey

Reply:
good question ... this is a change that came from the working group during mattercon, so I am not sure who introduced that ... I can just change the existing type, since the comment is valid! :D